client/src/js/components/bhDateInterval.js

```diff
@@ -0,0 +1,92 @@
+(function (angular) {
```

Comment:
Our `gulp` build will automatically do this wrapping.  There is not need to wrap your functions in a IIFE.